docs/technical-design/feature-brief-session-expiration.md

```diff
@@ -0,0 +1,38 @@
+# Feature brief: Session Expiration
+
+## Introduction
+
+We use a third-party authentication provider ([IDM](https://confluenceent.cms.gov/display/IDM/IDM+Trainings+and+Guides)) which automatically logs out sessions due to inactivity after about 30 minutes. We don't have direct access to their timekeeping, but we know this is about how long they allow a session to be inactive. Thus, we manually track sessions internally for the same time period.
+
```

Comment:
Should we add a sentence about what role LaunchDarkly plays in this feature? Like toggling the feature and configuring the session expiration time since both flags function almost as configuration instead of feature flagging. I know `Implementation details` has some details on it, but it would be nice to include a sentence here.